.eslintrc.js

```diff
@@ -5,7 +5,16 @@ module.exports = {
   rules: {
     // React Cosmos is designed to work with React >= 0.13 and stateless
     // functions were invented in 0.14
-    "react/prefer-stateless-function": 0
+    "react/prefer-stateless-function": 0,
+    // Dynamic require is needed in Playground
+    "import/no-dynamic-require": 0,
+    // Object and Array are forbidden by default too
+    "react/forbid-prop-types": [2, { "forbid": [ "any" ] }],
+    // Passing children as prop is needed in React Cosmos
+    "react/no-children-prop": 0,
```

Comment:
I'd add a TODO comment to fix this, with a link to your comment: https://github.com/react-cosmos/react-cosmos/issues/217#issuecomment-269708146